.gitignore

```diff
@@ -0,0 +1,75 @@
+# Created by .ignore support plugin (hsz.mobi)
```

Comment:
I'm surprised it pulled in all this stuff.  *.ear, *.war, etc.  We've used https://www.gitignore.io/ in the past because you can pick and choose what to pull in.